vendor/engines/secure_rooms/app/views/secure_rooms/card_readers/new.html.haml

```diff
@@ -0,0 +1,15 @@
+= content_for :h1 do
+  = current_facility
+= content_for :sidebar do
+  = render "admin/shared/sidenav_product", sidenav_tab: "secure_rooms"
+= content_for :tabnav do
+  = render "admin/shared/tabnav_product", secondary_tab: "card_readers"
+
+%h2= @product
+%h3= text("views.admin.secure_rooms.card_readers.new.title")
+
+= simple_form_for @card_reader, url: facility_secure_room_card_readers_path(current_facility, @product) do |f|
```

Comment:
I'm a little less concerned with the `simple_form_for` definition, but I've found that having the actual form fields be separate between new and edit to actually cost more in the long run because of how easy it is for them to get out of sync. But thanks for changing it.